src/state/slices/portfolioSlice/utils/index.ts

```diff
@@ -76,7 +77,7 @@ export const accountIdToLabel = (accountId: AccountId): string => {
       // TODO(0xdef1cafe): translations
       if (pubkey.startsWith('xpub')) return 'Legacy'
       if (pubkey.startsWith('ypub')) return 'Segwit'
-      if (pubkey.startsWith('zpub')) return 'Segwit Native'
+      if (pubkey.startsWith('zpub') || bech32.decode(pubkey).prefix === 'bc') return 'Segwit Native'
```

Comment:
might want to leave a todo that this won't be sufficient come taproot support since both start with `bc`

Reply:
87fe468f6d2bfa36ad4c25b82865cdb891b02ca4